apps/i18n/common/en_us.json

```diff
@@ -199,6 +199,7 @@
   "dialogCancel": "Cancel",
   "dialogConfirmEmail": "Confirm email",
   "dialogOK": "OK",
+  "dialogOkay": "Okay",
```

Comment:
Should we just use OK here? Seems weird to use OK sometimes and Okay other times